### PR TITLE
Include info about environment vars for gatsby config

### PR DESCRIPTION
**Pull request by njosefbeck**
Here you go! Let me know if there are any changes you'd like me to make to what I have :).

### CODE REVIEW

File: docs/docs/environment-variables.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: "Environment Variables"
 
 You can easily provide environment variables to your site. Just add a `.env.development` and/or `.env.production` file in your root folder for development or production builds respectively. The environment variables are embedded during build time using Webpack's [DefinePlugin](https://webpack.js.org/plugins/define-plugin/). Because these variables are provided at build time, you will need restart your dev server or rebuild your site after changing them.
 
+To add environment variables to your ```gatsby-config.js```, you can use the NPM package [dotenv](https://www.npmjs.com/package/dotenv). Once you've installed dotenv and followed their setup instructions, you can use your environment variables in the same way as shown in the example below.
```
Comment (KyleAMathews) [inline docs/docs/environment-variables.md]: You only need one backtick around inline code in markdown.

Also, let's make clearer the distinction between *browser* and *node* code. So break up the previous paragraph at "The environment variables". And preface that new paragraph with "For JavaScript loaded into the browser, the environment…"

Then for your new paragraph, preface it with "For JavaScript run in node.js e.g. in `gatsby-config.js` and `gatsby-node.js`, to add environment variables, you can use the NPM…"

Or something like that :-)

**Comment by njosefbeck** [comment]
I've amended the wording based on your suggestions! Let me know if that works for you :).

**Comment by KyleAMathews** [comment]
Looks good! Thanks for jumping in from our twitter discussion :-)

**Comment by jlengstorf** [comment]
Hiya @njosefbeck! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**